networkdb/cluster.go

```diff
@@ -449,8 +453,9 @@ func (nDB *NetworkDB) bulkSyncNode(networks []string, node string, unsolicited b
 	// Wait on a response only if it is unsolicited.
 	if unsolicited {
 		startTime := time.Now()
+		t := time.NewTimer(30 * time.Second)
 		select {
-		case <-time.After(30 * time.Second):
```

Comment:
`time.After` consumes much more memory that `Timer`. If you don't expect a timeout to happen a lot then you use the `Timer` then `time.After`